app/templates/_travis.yml

```diff
@@ -79,8 +79,8 @@ script:
   - casperjs test dashboard.js
 
   # Upload failed images to imgur
-  # - bash $TRAVIS_BUILD_DIR/imgur ./failures/login-form_0.fail.png
-  # - bash $TRAVIS_BUILD_DIR/imgur ./failures/authors-list_1.fail.png
+  - sleep 5
+  - bash scan_phantomcss_fails.sh > ~/phantomcss.log 2>&1 &
```

Comment:
@amitaibu I wanted it to print the list of images, It didn't without it, but i will check again, because i changed the script a bit.
I had a problem with the directory. that it was not created. I will make a test tomorrow morning to make sure i can get rid of them.